Modix.Data/Models/Core/MessageEntity.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Modix.Data.Models.Core
+{
+    public class MessageEntity
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public long Id { get; set; }
+
+        [Required]
+        public ulong MessageId { get; set; }
+
+        [Required]
+        public ulong GuildId { get; set; }
+
+        [Required]
+        public ulong ChannelId { get; set; }
+
+        [Required]
+        public ulong UserId { get; set; }
+
+        [Required]
+        public DateTimeOffset Timestamp { get; set; }
+
+        [Required]
+        public string OriginalMessageHash { get; set; }
```

Comment:
Is string the most appropriate type for the hash? Is there a hashing algorithm we could use that spits out a `long`?

Reply:
This seemed like the most straightforward approach. What advantage would a long offer over a string?

Reply:
Aside from being smaller, not much I guess. It just struck me as slightly odd. Maybe change the name to make it clear that that's a base64 encoding? Or make it a `byte[]`?